app/controllers/appointments_controller.rb

```diff
@@ -0,0 +1,23 @@
+# JSON API controller that serves Sections resource
+class AppointmentsController < ApplicationController
+  respond_to :json
+  before_filter :assign_model
+  after_action :verify_authorized, except: :index
+
+  def assign_model
+    @model = EmploymentPolicy::Scope
+      .new(current_user, Employment).resolveAppointments
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.